{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:40:39"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:40:39"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:43:07"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:43:07"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:45:14"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:45:14"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:47:01"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:47:01"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:48:37"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:48:37"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:50:29"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:50:29"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:50:47"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:50:47"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:51:58"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:51:58"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:52:49"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:52:49"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:54:37"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:54:37"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:56:24"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:56:24"}
{"level":"error","message":"Unable to connect to the database: connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:56:30"}
{"level":"error","message":"connect ECONNREFUSED 127.0.0.1:5432","name":"SequelizeConnectionRefusedError","original":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"parent":{"address":"127.0.0.1","code":"ECONNREFUSED","errno":-111,"port":5432,"syscall":"connect"},"service":"user-service","stack":"SequelizeConnectionRefusedError: connect ECONNREFUSED 127.0.0.1:5432\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:133:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorEvent (/usr/src/app/node_modules/pg/lib/client.js:337:19)\n    at Connection.emit (events.js:400:28)\n    at Socket.reportStreamError (/usr/src/app/node_modules/pg/lib/connection.js:58:12)\n    at Socket.emit (events.js:400:28)\n    at emitErrorNT (internal/streams/destroy.js:106:8)\n    at emitErrorCloseNT (internal/streams/destroy.js:74:3)\n    at processTicksAndRejections (internal/process/task_queues.js:82:21)","timestamp":"2024-06-14 14:56:30"}
{"level":"error","message":"Unable to connect to the database: no PostgreSQL user name specified in startup packet","name":"SequelizeConnectionError","original":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"parent":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"service":"user-service","stack":"SequelizeConnectionError: no PostgreSQL user name specified in startup packet\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/usr/src/app/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (events.js:400:28)\n    at /usr/src/app/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (events.js:400:28)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)","timestamp":"2024-06-14 14:57:51"}
{"level":"error","message":"no PostgreSQL user name specified in startup packet","name":"SequelizeConnectionError","original":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"parent":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"service":"user-service","stack":"SequelizeConnectionError: no PostgreSQL user name specified in startup packet\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/usr/src/app/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (events.js:400:28)\n    at /usr/src/app/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (events.js:400:28)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)","timestamp":"2024-06-14 14:57:51"}
{"level":"error","message":"Unable to connect to the database: no PostgreSQL user name specified in startup packet","name":"SequelizeConnectionError","original":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"parent":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"service":"user-service","stack":"SequelizeConnectionError: no PostgreSQL user name specified in startup packet\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/usr/src/app/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (events.js:400:28)\n    at /usr/src/app/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (events.js:400:28)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)","timestamp":"2024-06-14 14:59:16"}
{"level":"error","message":"no PostgreSQL user name specified in startup packet","name":"SequelizeConnectionError","original":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"parent":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"service":"user-service","stack":"SequelizeConnectionError: no PostgreSQL user name specified in startup packet\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/usr/src/app/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (events.js:400:28)\n    at /usr/src/app/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (events.js:400:28)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)","timestamp":"2024-06-14 14:59:16"}
{"level":"error","message":"Unable to connect to the database: no PostgreSQL user name specified in startup packet","name":"SequelizeConnectionError","original":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"parent":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"service":"user-service","stack":"SequelizeConnectionError: no PostgreSQL user name specified in startup packet\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/usr/src/app/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (events.js:400:28)\n    at /usr/src/app/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (events.js:400:28)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)","timestamp":"2024-06-14 15:01:17"}
{"level":"error","message":"no PostgreSQL user name specified in startup packet","name":"SequelizeConnectionError","original":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"parent":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"service":"user-service","stack":"SequelizeConnectionError: no PostgreSQL user name specified in startup packet\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/usr/src/app/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (events.js:400:28)\n    at /usr/src/app/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (events.js:400:28)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)","timestamp":"2024-06-14 15:01:17"}
{"level":"error","message":"Unable to connect to the database: no PostgreSQL user name specified in startup packet","name":"SequelizeConnectionError","original":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"parent":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"service":"user-service","stack":"SequelizeConnectionError: no PostgreSQL user name specified in startup packet\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/usr/src/app/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (events.js:400:28)\n    at /usr/src/app/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (events.js:400:28)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)","timestamp":"2024-06-14 15:02:07"}
{"level":"error","message":"no PostgreSQL user name specified in startup packet","name":"SequelizeConnectionError","original":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"parent":{"code":"28000","file":"postmaster.c","length":121,"line":"2287","name":"error","routine":"ProcessStartupPacket","severity":"FATAL"},"service":"user-service","stack":"SequelizeConnectionError: no PostgreSQL user name specified in startup packet\n    at Client._connectionCallback (/usr/src/app/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:145:24)\n    at Client._handleErrorWhileConnecting (/usr/src/app/node_modules/pg/lib/client.js:327:19)\n    at Client._handleErrorMessage (/usr/src/app/node_modules/pg/lib/client.js:347:19)\n    at Connection.emit (events.js:400:28)\n    at /usr/src/app/node_modules/pg/lib/connection.js:117:12\n    at Parser.parse (/usr/src/app/node_modules/pg-protocol/dist/parser.js:36:17)\n    at Socket.<anonymous> (/usr/src/app/node_modules/pg-protocol/dist/index.js:11:42)\n    at Socket.emit (events.js:400:28)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)","timestamp":"2024-06-14 15:02:07"}
